devicetypes/smartthings/motion-detector.src/motion-detector.groovy

```diff
@@ -34,14 +35,34 @@ metadata {
 				attributeState("active", label:'motion', icon:"st.motion.motion.active", backgroundColor:"#00A0DC")
 				attributeState("inactive", label:'no motion', icon:"st.motion.motion.inactive", backgroundColor:"#CCCCCC")
 			}
- 		}
+		}
 		main "motion"
 		details "motion"
 	}
 }
 
+def installed() {
```

Comment:
you might want to add something here to poll the device for initial status since you mentioned you're having problems

Also do you want to add a ping method?

Reply:
That's a good idea, but i'm not sure about adding a ping method since it's a sleepy device. Anyway, let's try that.

Reply:
Okay. Didn't realize it was sleepy. If it's sleepy you can't ping it so that's fine, but you should definitely try to get initial state before the device goes to sleep during the add process.